script/start-test-environment.sh

```diff
@@ -8,7 +8,7 @@ ROOTDIR=`cd $(dirname $SCRIPT)/.. && pwd`
 
 COMMAND="${@:-bash}"
 
-if ! minikube version | grep -q "v0.22.3"; then
+if ! minikube version | grep -q "v0.24.1"; then
```

Comment:
FWIW I would just remove the check now

Reply:
I think its okay to have this check. I almost wasted a day where travis CI was failing but integration tests were passing locally. Becuase i was using minikube 0.24.1 locally. Travis was using minukube 0.24.1. Minikube 0.22.2 has issues with kubernetes 1.8.0. This is just e.g, it better run same environment as what is run on travis.